expand_confidential_bundle.sh

```diff
@@ -8,11 +8,12 @@ set -o nounset
 set -o xtrace
 
 zipArchive=$1
+zipFolder=$2
```

Comment:
Is this and the changes starting on line 13 to handle when you download the zip bundle by clicking on the folder?

I ran into this with Ryan, and we ended up downloading all the files from within the folder.

Reply:
Yes this is for that case, without it the unzip was looking in the wrong place.

Reply:
Do you always need to download the folder as a whole now? (not a problem, just clarifying)

Reply:
You can also download the files individually, but downloading the folder of the files is easier.

Reply:
ok, cool. But do add an example of usage in the comments above